database/gormdb_test.go

```diff
@@ -466,9 +467,9 @@ func TestGormDBAssociateUserWithRemoteIdentity(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	var user1 pb.User
+	var gotUser1 pb.User
```

Comment:
Suggest to do `gotUser1 := &pb.User{}` instead to avoid passing `gotUser1` as value to the `cmp.Diff` function as explained in a previous comment (linter complains).